scripts/requirements.txt

```diff
@@ -107,8 +107,6 @@ safe-pysha3==1.0.4
 six==1.16.0
 sniffio==1.3.0
 stack-data==0.6.2
-terra-proto==4.0.2
```

Comment:
Can you comment these our so if we need to deploy to terra we know the imports required